src/main/resources/io/jenkins/plugins/designlibrary/Dialogs/index.jelly

```diff
@@ -0,0 +1,76 @@
+        <s:code language="java" file="alert.js" code="showAlert()" executable="true"/>
+      </l:tabPane>
+      <l:tabPane title="${%Prompts}">
+        <p class="jdl-paragraph">${%prompts}</p>
+        <s:code language="java" file="prompt.js" code="showPrompt()" executable="true"/>
+      </l:tabPane>
+      <l:tabPane title="${%Confirmations}">
+        <p class="jdl-paragraph">${%confirmations}</p>
+        <s:code language="java" file="confirm.js" code="showConfirm()" executable="true"/>
+      </l:tabPane>
+      <l:tabPane title="${%Modals}">
+        <p class="jdl-paragraph">${%modals}</p>
+        <s:code language="java" file="modal.js" code="showModal()" executable="true"/>
+        <s:code language="xml" file="modal.jelly"/>
+      </l:tabPane>
+    <l:tabPane title="${%Forms}">
+      <p class="jdl-paragraph">${%forms}</p>
+      <br/><br/>
+      <f:toggleSwitch id="formsubmit" checkedTitle="${%Submit form}" checked="true"
+                      title="${%Handle form in javascript}."
+      />
+      <s:code language="java" file="form.js" code="showForm()" executable="true"/>
+      <s:code language="xml" file="form.jelly"/>
+    </l:tabPane>
+    <hr/>
+    <div>
+      <h2>Customizing the appearance and behaviour</h2>
+      All dialogs take a second optional parameter with options that allow to change certain aspects.
+      <ul>
+        <li><code>title</code>: Adds a title to the dialog</li>
+        <li><code>okText</code>: Adjust the text of the OK button, defaults to <code>OK</code> or <code>Yes</code></li>
+        <li><code>cancelText</code>: Adjust the text of the Cancel button, defaults to <code>Cancel</code></li>
+        <li><code>type</code>: Change the color of the <code>OK</code> button. Allowed values: <code>destructive</code></li>
+        <li><code>minWidth</code>: Set the minimum width of the dialog, defaults to <code>300px</code></li>
+        <li><code>maxWidth</code>: Set the maximum width of the dialog, defaults to <code>475px</code></li>
+        <li><code>hideCloseButton</code>: Hides the close button in modal dialogs, defaults to <code>false</code></li>
+        <li><code>allowEmpty</code>: When set to <code>false</code> the <code>OK</code> button will be disabled in the
+          prompt dialog as long as the input field is empty or contains only whitespace, defaults to <code>false</code></li>
+        <li><code>submitButton</code>: Change the behaviour of the submit button in a form dialog. When <code>true</code>
+          the form will be submitted to the <code>action</code> attribute of the form. When <code>false</code>, a
+          <code>FormData</code> object is passed to the <code>resolve</code> method of the <code>Promise</code>.
+          Defaults to <code>true</code></li>
+      </ul>
+    </div>
+    <st:adjunct includes="io.jenkins.plugins.designlibrary.Dialogs.dialogs"/>
+    <script src="${resURL}/jsbundles/section-to-tabs.js" type="text/javascript" />
+  </s:sample>
+</j:jelly>
```

Comment:
This requires a SNAPSHOT version of Jenkins to see that in action? For me the execute button simply throws an error.

Reply:
I deployed the plugin into my latest Jenkins instance.

Reply:
if you run this with `mvn hpi:run` then you will get the incremental version automatically